### PR TITLE
Added a Volunteer Permission Level

### PR DESCRIPTION
**Pull request by sarj21**
We don't want volunteers being able to edit people or see stats or send mass emails so this should allow them to only see users and check them in

### CODE REVIEW

**Comment by mecolmg** [comment]
We should also hide the checkbox and the link button (see below)
image

**Comment by sarj21** [comment]
Oh I should hide the checkbox and also check that it bounces volunteers. I think I want to leave the link button so volunteers can pull up a persons details if they need to. I might just want to scrub the results of some stuff they dont need to see. We can ask the others what they think

**Comment by sarj21** [comment]
This seems to work now. The checkboxes are hidden from volunteers and they can't hit any important admin only pages. Also, I stepped them down to demi-gods

File: app/client/views/admin/users/users.html [hunk 1]
```diff
@@ -97,7 +97,7 @@ <h3> {{section.title}} </h3>
         class="ui celled selectable compact users table">
         <thead>
           <tr>
-            <th></th>
+            <th ng-if="selectedUser.admin"></th>
```
Comment (mecolmg) [inline app/client/views/admin/users/users.html]: Should be `currentUser.admin`

This currently breaks admin view normally.

**Review by mecolmg — CHANGES_REQUESTED** [review]
Also, the volunteers currently don't have access to the `user` page that the little link button goes to, mainly because that page allows updating user info. Volunteers can see user info by clicking on the row, so there's no reason for them to see the link.

**Comment by sarj21** [comment]
Yeah my confusion, I forgot what the link button did. And I used the wrong user source. It looks like its fixed

**Review by mecolmg — APPROVED** [review]

**Pull request merged**